packages/suite/src/views/wallet/transactions/components/InfoCard.tsx

```diff
@@ -2,18 +2,12 @@ import { ReactNode } from 'react';
 
 import styled from 'styled-components';
 
-import { variables, Card, SkeletonRectangle } from '@trezor/components';
+import { variables, Card, SkeletonRectangle, Column } from '@trezor/components';
 import { BigNumber } from '@trezor/utils/src/bigNumber';
 
 import { HiddenPlaceholder, FormattedCryptoAmount, Sign } from 'src/components/suite';
 import { Account } from 'src/types/wallet';
 
-const InfoCardContent = styled.div`
-    display: flex;
-    width: 100%;
-    flex-direction: column;
-`;
-
 const Title = styled.div`
```

Comment:
Can you please also remove all of these custom styles? They are easily replaced with components